### PR TITLE
Add load-test for mutation and subscription

### PR DESCRIPTION
**Pull request by MohamedSabthar**
## Purpose

Fixes: https://github.com/ballerina-platform/ballerina-standard-library/issues/3562
Related PR: https://github.com/ballerina-platform/ballerina-performance-cloud/pull/11671

## Checklist
- [x] Linked to an issue
- [ ] ~~Updated the changelog~~
- [x] Added tests
- [ ] ~~Updated the spec~~
- [ ] ~~Checked native-image compatibility~~

### CODE REVIEW

File: load-tests/book_reviews_mutation/src/modules/datasource/types.bal [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+public type ReviewData record {|
+    string bookId;
+    string author;
+    string comment;
+|};
+
+public type ReviewRow record {
+    readonly string id;
+    *ReviewData;
+};
```
Comment (ThisaruGuruge) [inline load-tests/book_reviews_mutation/src/modules/datasource/types.bal:36]: Can't we make these `readonly & record` types? Then we don't need the `cloneReadOnly` methods.

File: load-tests/book_reviews_mutation/src/book.bal [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+// specific language governing permissions and limitations
+// under the License.
+
+import book_reviews_mutation.datasource as ds;
+
+# Represents a book.
+public isolated distinct service class Book {
+    private final readonly & ds:BookRow bookRow;
+
+    function init(ds:BookRow bookRow) {
```
Comment (ThisaruGuruge) [inline load-tests/book_reviews_mutation/src/book.bal:23]: We can make `ds:BookRow` a readonly type.

File: load-tests/book_reviews_mutation/src/modules/datasource/datasource.bal [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+];
+
+isolated final table<ReviewRow> key(id) reviews = table [
+    {id: "1", author: "John", comment: "Good", bookId: "1"},
+    {id: "2", author: "Peter", comment: "Excellent", bookId: "1"},
+    {id: "3", author: "John", comment: "Good", bookId: "2"},
+    {id: "4", author: "Peter", comment: "Excellent", bookId: "2"}
+];
+
+isolated int lastReviewId = 4;
```
Comment (ThisaruGuruge) [inline load-tests/book_reviews_mutation/src/modules/datasource/datasource.bal:29]: Why do we need this? Can't we use an auto-increment id (or maybe an auto-generated UUID) for this?

File: load-tests/book_reviews_mutation/src/service.bal [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    # A simple resource that returns a welcome message
+    # + return - greeting message
+    isolated resource function get geeting() returns string {
+        return "Welcome to Book Review Service";
+    }
+
+    # Add review to a book
+    # + input - represents a review input
+    # + return - the newly added Review
+    isolated remote function reviewAdd(readonly & ReviewInput input) returns Review|error {
```
Comment (ThisaruGuruge) [inline load-tests/book_reviews_mutation/src/service.bal:37]:
```suggestion
    isolated remote function addReview(readonly & ReviewInput input) returns Review|error {
```

File: load-tests/book_reviews_mutation/src/service.bal [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+        ds:ReviewRow reviewRow = check ds:addReview(input);
+        check self.pubsub.publish("Reviews", reviewRow.cloneReadOnly(), timeout = -1);
```
Comment (ThisaruGuruge) [inline load-tests/book_reviews_mutation/src/service.bal:39]: If we make the review a readonly type, we can omit this `cloneReadOnly` call as well.

File: load-tests/book_reviews_mutation/src/modules/datasource/datasource.bal (hunk 3 above)
Comment (ThisaruGuruge) [reply load-tests/book_reviews_mutation/src/modules/datasource/datasource.bal:29]: I think we can use the [`nextKey` function](https://lib.ballerina.io/ballerina/lang.table/0.0.0#nextKey) for this.

Comment (MohamedSabthar) [reply load-tests/book_reviews_mutation/src/modules/datasource/datasource.bal]: Updated with UUID

File: load-tests/book_reviews_mutation/src/service.bal (hunk 4 above)
Comment (MohamedSabthar) [reply load-tests/book_reviews_mutation/src/service.bal]: fixed with [f0d16cd](https://github.com/ballerina-platform/module-ballerina-graphql/pull/1308/commits/f0d16cde2bb76ab77b471ccecb4605c4e7d74074)

File: load-tests/book_reviews_mutation/src/service.bal (hunk 5 above)
Comment (MohamedSabthar) [reply load-tests/book_reviews_mutation/src/service.bal]: made types readonly with [2d4ed9a](https://github.com/ballerina-platform/module-ballerina-graphql/pull/1308/commits/2d4ed9a102749ec104813d2c82b2e4f214053bfd)

**Pull request merged**